server/tests/phase.service.test.js

```diff
@@ -64,4 +66,102 @@ describe('Phase Allocation Endpoints', () => {
     const res = await getAllSitePhases(site.id);
     expect(res.length).not.toEqual(0);
   });
+
+  it('getAllSitePhases, returns correct # of hires and remaining hires per phase', async () => {
```

Comment:
Should probably be phrased more like "calls getAllSitePhases, ...", although formatting isn't consistent here either way. This can be left alone for now.